src/database/DataTypes/Abbreviation.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * mSupply Mobile
+ * Sustainable Solutions (NZ) Ltd. 2016
```

Comment:
should these be 2019 now?

Reply:
haha whoopsies, copy pasted the wrong file.. should they all be changed in the entire code base? And if so, maybe I should just change everything to 2020  ?